### ISSUE TITLE
Issue with GetOrgs

### ISSUE THREAD

**Issue by sherodkeen** [Ed-Fi-Exchange-OSS/Ed-Fi-Clever-Integration#1]
I have been able to get the application to build and run successfully using VS 2019. I was able to POST to /OAuth/token using the default dev config in the swagger page with success, recieving a 200 response and an access_token. However, I keep getting 401's both in Swagger and Postman. Any troubleshooting ideas?

**Comment by dougloyo** [comment]
Hi, @sherodkeen 

Great to hear from you!
So the short answer is that you have to use OAuth1 to validate your calls. You can review the spec here: https://datatracker.ietf.org/doc/html/rfc5849
The creation of the oauth_signature is really interesting. look at: https://datatracker.ietf.org/doc/html/rfc5849#section-3.4.1.1

If you use Postman its easier to consume as they have a implemented OAuth1 in their authorization providers.
![image](https://user-images.githubusercontent.com/573569/120714721-309b9b80-c489-11eb-8baf-a8d4a76ffd75.png)

**Comment by sherodkeen** [comment]
@dougloyo Perfect!!  Thank you for giving the documentation and the screenshot. Nice to see the official documentation on OAuth 1.0 and how the signature should be formed. The screenshot helped me to get it right in Postman. The default port in the Postman Environment was to point to port 3001. When I looked in Swagger at the /OAuth/token endpoint, it was using port 44354. In addition to making my authorization in postman match the screenshot settings, I had to update the apiRoot Environment variable from localhost:3001 to https://localhost:44354. (See screenshot). Thanks so much for making this Ed-Fi to Clever tool and for your help getting it stood up!
![Snag_33852b00](https://user-images.githubusercontent.com/4204319/120803287-64220880-c511-11eb-8663-8337ce530c3c.png)
